### ISSUE TITLE
Script running iframe only (not in the "real" page)

### ISSUE THREAD

**Issue by matheusfaustino** [violentmonkey/violentmonkey#394]
### What is the problem?
This script is running on this page (https://www.hltv.org/matches/2322013/gambit-vs-big-esl-one-belo-horizonte-europe-closed-qualifier) inside the iframe only. 
```// ==UserScript==
// @name Direct Link stream HLTV
// @namespace hltv.direct.stream.link
// @match *://*/*
// ==/UserScript==

console.log('=========================================');
console.log(window, document,window.top);

window.top.document.querySelectorAll('.stream-box').forEach(e => {
  if (!e.hasAttribute('data-stream-embed')) return;
  
  let link = document.createElement('a');
  link.textContent = 'Direct Link ' + e.querySelector('.flagAlign').textContent;
  link.href = e.getAttribute('data-stream-embed');
  link.target = '_blank';
  e.parentNode.insertBefore(link, e.nextSibling);  
});
```

### How to reproduce it?
1. Open the page and look to the console

### What is the expected result?
You should see the object document from the page and it will create links for each stream link.
(it works on Greasemonkey 4.3)

### What is the actual result?
You will see the object document from the iframe

### Environment
- Browser: Firefox
- Browser version: 60.0b12 (64-bit)
- Violentmonkey version: 2.9.1
- OS: w10

**Comment by gera2ld** [comment]
The script is blocked by CSP.
duplicate of #385
